channel.c

```diff
@@ -100,6 +100,7 @@ static const char * const modifier_names[] = {
 	[IIO_MOD_CO2] = "co2",
 	[IIO_MOD_ETHANOL] = "ethanol",
 	[IIO_MOD_H2] = "h2",
+	[IIO_MOD_O2] = "h2",
```

Comment:
Is this a typo? shouldn't this be "o2" ?

Reply:
oh crap...
thank for catching
PR #609